TheKeyOAuth2/TheKeyOAuth2Client.m

```diff
@@ -156,9 +156,10 @@ -(void)ticketForServiceURL:(NSURL *)service complete:(void (^)(NSString *ticket)
 }
 
 -(TheKeyOAuth2Authentication *)newAuthenticationUsingKeychain:(BOOL)useKeychain {
+    NSString *redirectURL = [NSString stringWithFormat:@"%@%@", [[self serverURL] absoluteString], TheKeyOAuth2RedirectURI];
```

Comment:
just to double check, this isn't introducing a double slash, is it?

is `serverURL` `https://thekey.me/cas` or `https://thekey.me/cas/`?